esp/esp/program/modules/base.py

```diff
@@ -512,28 +512,56 @@ def _checkGrade(moduleObj, request, tl, *args, **kwargs):
 
 # Just broke out this function to allow combined deadlines (see meets_any_deadline,
 # meets_all_deadlines functions below).  -Michael P, 6/23/2009
+
+# Combined deadlines are annoying; meets_deadline has been absorbed into
+# meets_any_deadline and meets_all_deadlines has been deleted --bpchen,
+# 2017-02-17
 def _checkDeadline_helper(method, extension, moduleObj, request, tl, *args, **kwargs):
+    """
```

Comment:
Thanks for the docstring :-D !  Per [PEP8/PEP257](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings) multiline docstrings should have a one-line summary, then a blank line, then whatever else.  (Happy to make more specific suggestions about what to say, if helpful -- bug me on slack/zephyr/etc.)